### ISSUE TITLE
incorrect description in README and importer

### ISSUE THREAD

**Issue by bladeXue** [rubysamurai/purecss-sass#6]
1. in `README.md` where the [USAGE](https://github.com/rubysamurai/purecss-sass#usage) says:

```scss
@import 'purecss/forms-nr';
```

but there is no such file

2. in importer file `vendor/assets/stylesheets/_purecss.scss`, it misses a line:

```scss
@import 'purecss/base';
@import 'purecss/buttons';
@import 'purecss/forms';
@import 'purecss/grids';						// WHERE IS THIS LINE?
@import 'purecss/grids-responsive';
@import 'purecss/menus';
@import 'purecss/tables';
```

**Comment by rubysamurai** [comment]
Apologies, `README` is a bit outdated.

`purecss/forms-nr` (non-responsive forms) is identical to `purecss/forms` (responsive forms) but without media queries ([here is the diff](https://www.diffchecker.com/cLVcl0qw/) for both files), so it makes little sense to use `purecss/forms-nr`. Mention of `forms-nr` will be removed from the `README`.

`purecss/grids` (non-responsive grids) file uses different classes than `purecss/grids-responsive`, so we still include it in the package for legacy reasons, but it is not included by default, using `@import 'purecss';`, which includes only responsive components to follow best practices.

**Comment by bladeXue** [comment]
> Apologies, `README` is a bit outdated.
> 
> `purecss/forms-nr` (non-responsive forms) is identical to `purecss/forms` (responsive forms) but without media queries ([here is the diff](https://www.diffchecker.com/cLVcl0qw/) for both files), so it makes little sense to use `purecss/forms-nr`. Mention of `forms-nr` will be removed from the `README`.
> 
> `purecss/grids` (non-responsive grids) file uses different classes than `purecss/grids-responsive`, so we still include it in the package for legacy reasons, but it is not included by default, using `@import 'purecss';`, which includes only responsive components to follow best practices.

I've understood the USAGE in `purecss/forms-nr`, it's ok now. But for the part of the Grid, I noticed that there is no lines of `.pure-g` and `.pure-u` styles in `purecss/grids-responsive` so it can not works independently. Current importer files can not take effect independently without importing `purecss/grids`. Maybe?

**Comment by rubysamurai** [comment]
You are right. I will make a new release with a fix today. 
Thank you for taking time to narrow down this issue.

**Comment by rubysamurai** [comment]
It should be fixed now in [v3.0.1](https://github.com/rubysamurai/purecss-sass/releases/tag/v3.0.1).
